index.js

```diff
@@ -21,6 +21,15 @@ const {
 module.exports = function (log, indexesPath) {
   debug('indexes path', indexesPath)
 
+  let bitsetCache = new WeakMap()
+  let sortedCache = { ascending: new WeakMap(), descending: new WeakMap() }
+
+  log.since(() => {
```

Comment:
Have you tested migrate of a full log with this patch and without? I'm wondering what the overhead of this is when running a lot of data through the log.

Reply:
I'll check it